src/olympia/abuse/models.py

```diff
@@ -201,7 +201,7 @@ def notify_reporters(self, action_helper):
             is_appeal=False,
         )
         action_helper.notify_reporters(
-            reporter_abuse_reports=self.appellants.all(),
+            reporter_abuse_reports=self.reporter_appellants.all(),
```

Comment:
what is the difference between `appelants` and `reporter_appelants` or is this a rename?

Reply:
it's a rename because the back reference was named `appellants` but really it's not all appellants - it's just reporter appellants.